lib/modules/apostrophe-modal/public/css/components/modal.less

```diff
@@ -74,22 +83,28 @@
   }
 }
 // Modal Body ===================================
+
+.apos-modal-contents {
+  flex: 1;
+  min-height: 0;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: stretch;
+}
 .apos-modal
 .apos-modal-content
 {
   position: absolute;
-
-	// this is replaced dynamically with js
-  height: ~'calc(100vh - 100px - 189px - 82px)';     // 100vh - 100 (position) - 189 (header + breadcrumb) - 82px (footer bar)
+  height: 100%;
   width: 100%;
 
-  // background-color: @apos-white;
 	background-color: lighten(@apos-light, 5%);
 }
 
 .apos-modal-body
 {
-	height: 100%;
+  height: calc(100% - 63px);
```

Comment:
Where is this number coming from?

Reply:
This is the height of the modal's footer, which changes only on mobiles, and I think we don't really support mobiles, but we could use the resizeHeightContent to make it dynamic.

Reply:
Ok I found a better solution to remove it, moreover it fixes the modal's footer in IE 11 !